autosklearn/automl.py

```diff
@@ -634,17 +639,24 @@ def fit(
         self.InputValidator.fit(X_train=X, y_train=y, X_test=X_test, y_test=y_test)
         X, y = self.InputValidator.transform(X, y)
 
-        if X_test is not None:
+        if X_test is not None and y_test is not None:
             X_test, y_test = self.InputValidator.transform(X_test, y_test)
 
-        X, y = self.subsample_if_too_large(
-            X=X,
-            y=y,
-            logger=self._logger,
-            seed=self._seed,
-            memory_limit=self._memory_limit,
-            task=self._task,
-        )
+        # We don't support size reduction on pandas dataframes yet
```

Comment:
I wrote it as a comment in `reduce_dataset_siize_if_too_large`. Essentially it's possible but I didn't implement it yet (and it wasn't implemented before).

Essentially it's because `ndarray` have a uniform dtype where as pandas does not.

```python
    NOTE: limitations
    * Does not support dataframes yet
        -   Requires implementing column wise precision reduction
        -   Requires calculating memory usage
```

Reply:
Alright, makes sense. Could you please open an issue that we support this in the future?

Reply:
Done, this is recorded in issue #1278